### PR TITLE
Teams nav menu

### PR DESCRIPTION
**Pull request by sstrubberg**
Cleaned up the in-page nav menus. They should be responsive now.
Fixed the tile--xl to be work with small screens.

### CODE REVIEW

File: src/components/Tile/tile.scss [hunk 1]
```diff
@@ -207,9 +207,11 @@ a.bx--tile--clickable--dark .tile__link-icon svg {
 }
 
 .tile--xl a.bx--tile {
-  right: 1rem;
   bottom: 0;
   position: absolute;
+  @include breakpoint('lg') {
```
Comment (alisonjoseph) [inline src/components/Tile/tile.scss]: can we get a line break between 211 and 212, makes the sass easier to read 😸

File: src/components/Markdown/markdown.scss [hunk 2]
```diff
@@ -97,7 +97,7 @@
 
 .page-ul,
 .page-ol {
-  margin: 0 0 6rem $spacing-lg;
+  // margin: 0 0 6rem 0.8rem;
```
Comment (alisonjoseph) [inline src/components/Markdown/markdown.scss]: should this be removed?

File: src/content/teams/teams.md [hunk 3]
```diff
@@ -88,8 +88,8 @@ Language
 
 ### Expertise
 
-Education  
-Organization  
+Education
```
Comment (alisonjoseph) [inline src/content/teams/teams.md]: I think the spaces you took out here are needed for a line break

File: src/components/Markdown/markdown.scss (hunk 2 above)
Comment (sstrubberg) [reply src/components/Markdown/markdown.scss]: yeah, it was throwing the menu off the grid based on the spec.

**Pull request merged**